doc/functions/fhs-environments.xml

```diff
@@ -50,6 +50,58 @@
     </para>
    </listitem>
   </varlistentry>
+  <varlistentry>
+    <term>
+      <literal>automount</literal>
+    </term>
+    <listitem>
+      <para>
+        Set to <literal>true</literal> by default, this option will bind mount
+        all directories in the root filesystem that are not in the generated
+        FHS environment. For example, if the FHS environment generates only
+        the <literal>/bin</literal> and <literal>/lib</literal> directories,
+        then other directories like <literal>/home</literal>
+        and <literal>/tmp</literal> will be bind mounted to the namespace.
+
+        This is usually what you want, since the FHS environment generates
+        all standard directories which provide binaries or libraries to
+        your system.
+      </para>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>
+      <literal>chdir</literal>
+    </term>
+    <listitem>
+      <para>
+        Set to <literal>"$(pwd)"</literal> by default, this option determines
```

Comment:
This seems to be orthogonal to FHS emulation (and can be/is implemented in wrappers).